package.json

```diff
@@ -17,14 +17,18 @@
     "clean": "gulp clean"
   },
   "main": "modules/torso.js",
+  "peerDependencies": {
+    "jquery": "2.1.x",
+    "backbone": "1.2.x",
+    "underscore": "1.8.x",
+    "handlebars": "3.0.x"
+  },
   "dependencies": {
-    "jquery": "2.1.3",
-    "underscore": "1.8.2",
-    "backbone": "1.2.3",
     "backbone-nested": "2.0.4",
```

Comment:
Fortunately, backbone-nested and backbone.stickit don't bring in jquery, underscore, or backbone when installed. Because they use Bower.